src/main/java/com/google/sps/classes/SqlConstants.java

```diff
@@ -33,6 +33,15 @@
   public static final int QUESTION_INSERT_ASKERID = 3;
 
   // Constants used for inserting a follower.
+<<<<<<< HEAD
+  public static final int QUESTION_FETCH_MAXID_COLUMN = 1;
+  public static final int FOLLOWER_INSERT_QUESTIONID_COLUMN = 1;
+  public static final int FOLLOWER_INSERT_ASKERID_COLUMN = 2;
+
+  // Constants to update mentor evidence.
+  public static final int MENTOR_EVIDENCE_PARAGRAPH = 1;
+  public static final int MENTOR_EVIDENCE_USERID = 2;
+=======
```

Comment:
You should remove the head and <<<<<<< and ====== signs of the merge. If you run this, it should mark some compilation errors.

Reply:
Whoops, missed this merge conflict. Done!